test/alternator/test_cql_rbac.py

```diff
@@ -124,3 +125,110 @@ def test_login_false(dynamodb, cql):
         with new_dynamodb(dynamodb, role, key) as d:
             with pytest.raises(ClientError, match='UnrecognizedClientException.*login=false'):
                 d.meta.client.list_tables()
+
+# Quote an identifier if it needs to be double-quoted in CQL. Quoting is
+# *not* needed if the identifier matches [a-z][a-z0-9_]*, otherwise it does.
```

Comment:
> Quoting is *not* needed if the identifier matches...

So if we quote identifiers that don't need to be quoted, will the CQL still work? If that's so, then I'm thinking maybe we can just quote everything, which will simplify the code.

Reply:
I just realized that quoting specific symbols also makes sense, and the code wouldn't be that simpler, so forget it.

Reply:
You are right that we can unconditionally quote instead of maybe_quote.  I actually proposed this in the past in Scylla - https://github.com/scylladb/scylladb/pull/9920, but reviewers liked maybe_quote to stay. In this case, this is just a test, and as you said it doesn't really complicate much (just adds an "if") so I think it's fine.

Reply:
Yeah, thanks for clarifications.

Reply:
Quotes in CQL also imply a case-sensitive match.